README.md

```diff
@@ -196,6 +196,9 @@ Key:
   Kubernetes cluster, but everything they have in common will work.
 * `-` The Kubernetes cluster has features the java-client library can't use
   (additional API objects, etc).
+* `x` The Kubernetes cluster has no guarantees to support the API client of
```

Comment:
Can we change this to "is not tested" rather than "may simply break"

Reply:
IIUC, it's more than just "not tested". If an API version is deprecated, and two versions later it is removed, then that functionality will simply not work.

Reply:
Tried to keep clarity while softening the statement. PTAL.